src/utils/utils.js

```diff
@@ -315,4 +315,40 @@ export function connectAccountDataFilter(account){
         isConnected:account.isConnected,
         isConnecting:account.isConnecting,
     }
+}
+
+/**
+ * split special style
+ * @returns 
+ */
+ export function specialSplit(str) {
+    let startStr = '[['
+    let endStr = "]]"
+    let list = []
+    var index = str.indexOf(startStr); 
+    let lastIndex = -endStr.length
+    let specialIndex = -1
+    while (index !== -1) {
```

Comment:
caveat: the string must have at least one `[[...]]` pair or the output will be empty